### PR TITLE
Migrate mui v5

### PR DESCRIPTION
**Pull request by mayayunx**
### Summary of changes
- Migrate Material UI from v4 to v5 according to [Mui](https://mui.com/guides/migration-v4/)
- Fix styling and failed tests after the migration

### Context and reason for change
- Version Update from MUI

### How can the changes be tested
- run `yarn test:unit`, `yarn test:integration-ci`, `yarn test:e2e`
- test UI in app

### CODE REVIEW

File: src/Frontend/Components/AttributionColumn/__tests__/attribution-column-helpers.test.ts [hunk 1]
```diff
@@ -12,11 +12,11 @@ import {
 
 describe('The AttributionColumn helpers', () => {
   test('getLicenseTextMaxRows in audit view', () => {
-    expect(getLicenseTextMaxRows(1080, View.Audit)).toEqual(35);
+    expect(getLicenseTextMaxRows(1080, View.Audit)).toEqual(29);
```
Comment (nicarl) [inline src/Frontend/Components/AttributionColumn/__tests__/attribution-column-helpers.test.ts:15]: I think default fontsize or the font itself changed? I am fine with it

**Review by nicarl — COMMENTED** [review]
Nice work!
I just found two small styling problems:

The search bar got quite a lot bigger as well as the tabs look a bit off at the moment.
Pictures for comparison

Before:
Screenshot_20211125_161330_old
Screenshot_20211125_161350_old

After:
Screenshot_20211125_161036
Screenshot_20211125_161059

File: src/Frontend/Components/AttributionColumn/__tests__/attribution-column-helpers.test.ts (hunk 1 above)
Comment (mayayunx) [reply src/Frontend/Components/AttributionColumn/__tests__/attribution-column-helpers.test.ts:15]: The height of one line in text area is changed from 16px to 19px.

**Review by nicarl — APPROVED** [review]
Thank you, very nice :sparkles:

**Review by leslielazzarino — APPROVED** [review]

**Pull request merged**